generators/app/templates/_.npmignore

```diff
@@ -1,5 +1,4 @@
 *.log
-node_modules
```

Comment:
Don't include what npm does by default: https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package